en/docs/manage-databases-and-caches/add-choreo-managed-databases-and-caches-to-the-marketplace.md

```diff
@@ -7,9 +7,9 @@ When you create a Choreo-managed database or cache, you can add it to the Market
 
 ## Step 1: Register credentials
 
-When you create a database or cache server, you get super admin credentials by default. You can use these credentials to create new user credentials. The steps to create new user credentials depend on the type of database or cache you are using.
+You can register either the default super admin credentials or credentials you create using the super admin credentials. When you create a database or cache server, you get super admin credentials by default. You can use these super user credentials to create new user credentials. The steps to create new user credentials depend on the type of database or cache you are using.
```

Comment:
Suggestion:
To register credentials, you can either use the default super admin credentials or the credentials you create using the super admin credentials. When you create a database or cache server, super admin credentials are generated by default. You can use these super admin credentials to create new user credentials. The steps to create new user credentials depend on the type of database or cache you are using.